UnitTests/BraintreeThreeDSecureTests/BTThreeDSecurePostalAddress_Tests.swift

```diff
@@ -1,85 +0,0 @@
-import XCTest
-@testable import BraintreeThreeDSecure
-
-class BTThreeDSecurePostalAddress_Tests: XCTestCase {
```

Comment:
It looks like we may have missed these test cases. We have `shippingBillingName` but not `givenName` and `billingGivenName` from this test class.

Reply:
Yep, you're right, looks like we always append only `billing` or `shipping`. Thanks for adding that to the test!